lib/win32/screenshot/bitmap_maker.rb

```diff
@@ -74,14 +102,34 @@ def capture_area(hwnd, context, x1, y1, x2, y2)
                   54
           ].pack('SLSSL')
 
-          Image.new(bmFileHeader + bmInfo + lpvpxldata.read_string(bitmap_size), w, h)
+          Image.new(bmFileHeader + bmInfo + lpvpxldata.read_string(bitmap_size), width, height)
         ensure
           lpvpxldata.free
           delete_object(hmemBM)
           delete_dc(hmemDC)
           release_dc(0, hScreenDC)
         end
 
+        def desktop_dimensions
```

Comment:
Maybe extract desktop related methods into a separate Struct type class:
```
# desktop.rb

Win32::Screenshot::Desktop = Struct.new(:left, :top, :width, :height) do
  def dimensions
    [left, top, width, height]
  end
end
```

Reply:
Done. Although the test for capturing desktop lost a little clarity: [link](https://github.com/bensandland/win32screenshot/blob/master/spec/win32/screenshot/take_spec.rb#L34)

Is it worthwhile to add getter functions to for width and height specifically?

Reply:
Have this method return `Desktop` Struct instead of its dimensions. Then whoever needs dimensions can do `desktop.dimensions` and whoever needs height/width or whatever from the Struct can do just like that (e.g `desktop.height` or `desktop.width`. Struct already has getters for all of its fields.

Reply:
Of course rename this method to `desktop` too instead of `desktop_dimensions`.

Reply:
Ah yes of course, that makes sense. Changed it now